src/Types.jl

```diff
@@ -906,7 +931,21 @@ function clone_or_cp_registries(ctx::Context, regs::Vector{RegistrySpec}, depot:
         end
         # clone to tmpdir first
         tmp = mktempdir()
-        if reg.path !== nothing # copy from local source
+        if (url = pkg_server_registry_url(reg.uuid)) !== nothing
```

Comment:
Seems like the registry UUID can be `nothing` (from the tests).

Reply:
👍 Addressed by adding a `pkg_server_registry_url(::Nothing) = nothing` method.